### ISSUE TITLE
ERROR: _reactNativeMeteor2.default.itemSubscribe is not a function

### ISSUE THREAD

**Issue by dgroch** [inProgress-team/react-native-meteor#1]
I've successfully authenticated via loginWithEmail

However, when I try to subscribe a publication, I get the error
`_reactNativeMeteor2.default.itemSubscribe is not a function`

Any ideas?
This is my code

```
this.messages = meteor.subscribe('messages', function(messages) {
        self.setState({
          dataSource: self.state.dataSource.clonewithRows(messages),
          loaded: true
        })
      });
```

**Comment by Mokto** [comment]
Hi,

1) You do not use itemSuscribe in your code, right ?
2) Can you please update this plugin to 0.3.2 so I can help you more efficiently ?

Thank you.

**Comment by Mokto** [comment]
Ok I just understood your error.

It was a miss spelling of the su**B**scribe word. It is now fixed in 0.5.0 ;)

**Comment by dgroch** [comment]
Thanks!!
